src/gateways/createArchiveLoader.ts

```diff
@@ -9,6 +11,30 @@ interface Extras {
   }>;
 }
 
+const cleanArchiveResponse = (archiveContent: any) => {
+  const chapterSlug = 'chapter-';
+  const chapterTitle = 'Chapter';
+  const appendixSlug = 'appendix-';
+  const appendixTitle = 'Appendix ';
+
+  if (archiveContent.tree) {
+    archiveContent.tree.contents.map((item: any) => {
+      if (item.slug.includes(chapterSlug) || item.slug.includes(appendixSlug)) {
+        item.slug = item.slug.replace(chapterSlug, '').replace(appendixSlug, '');
+      }
+
+      const splitTitle = splitTitleParts(item.title);
+
+      if (!isNullOrUndefined(splitTitle[0]) && (splitTitle[0].includes(chapterTitle) ||
+          splitTitle[0].includes(appendixTitle))) {
+        item.title = item.title.replace(chapterTitle, '').replace(appendixTitle, '');
```

Comment:
this makes me nervous, since we cant limit it to just the start of the string, what if the rest of the title has the bad word in it, when we only want to replace it in the `.os-number` section. you might have to parse this with DOMParser so that you can manipulate just that one section and then render it back to text